src/nvim/globals.h

```diff
@@ -977,6 +977,8 @@ EXTERN const char bot_top_msg[] INIT(= N_("search hit BOTTOM, continuing at TOP"
 EXTERN const char line_msg[] INIT(= N_(" line "));
 
 EXTERN FILE *time_fd INIT(= NULL);  // where to write startup timing
+#define STARTUP_TIME_BUF_SIZE 8192
+EXTERN char *startup_time_buf INIT(= NULL); // startup timing buffer
```

Comment:
Avoid unnecessary entropy. The option is named `--startuptime`.
```suggestion
EXTERN char *startuptime_buf INIT(= NULL); // --startuptime buffer
```